authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.2 on 2018-11-05 08:37
+# Generated by Django 2.1.2 on 2018-11-07 04:36
```

Comment:
Why are you updating this?

Reply:
It is automatically updated when you run migrations